test/rules/import-group-order/test.10.tsx.lint

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import { RandomGroup as TRandomGroup, RandomGroupId } from 'stores/RandomStore';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Named imports are not sorted in alphabetical order]
+
+import { createMagic } from 'constants/magic';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Import convention has been violated. This is auto-fixable.]
```

Comment:
What's wrong with this one?

Reply:
This is actually nasty hack we are using. This line is there every time the issue is autofixable – that's because the fixer is connected to this line and the fixer works that he rewrite all the imports at once, not issue per issue. * shy boi *

Reply:
Ah I see! Okay makes sense, thanks 🎂